src/Esfa.Vacancy.Api.Types/ApprenticeshipSummary.cs

```diff
@@ -92,6 +92,11 @@ public class ApprenticeshipSummary
         /// </summary>
         public string ApiDetailUrl { get; set; }
 
+        /// <summary>
+        /// Is employer disability confident.
+        /// </summary>
+        public bool IsEmployerDisabilityConfident { get; set; }
```

Comment:
In terms of context, I am thinking 'IsDisabilityConfidentEmployerVacancy' may suit better, I know the issue this causes because of what it has been called in the DB.

Reply:
If we say "Is the employer disability confident about this vacancy" then it works for both informing and questioning. Where "this vacancy" is implied by the vacancy object in response or in parameter.